test/text.tsx

```diff
@@ -40,25 +40,25 @@ test('text with rgb color', t => {
 	t.is(output, chalk.rgb(255, 136, 0)('Test'));
 });
 
-test('text with hsl color', t => {
-	const output = renderToString(<Text color="hsl(32, 100, 50)">Test</Text>);
-	t.is(output, chalk.hsl(32, 100, 50)('Test'));
-});
+// test('text with hsl color', t => {
+// 	const output = renderToString(<Text color="hsl(32, 100, 50)">Test</Text>);
+// 	t.is(output, chalk.hsl(32, 100, 50)('Test'));
+// });
```

Comment:
^

Reply:
`chalk.hsl` no longer exists. What should I be using instead?